pygeo/DVGeometry.py

```diff
@@ -1301,6 +1409,10 @@ def update(self, ptSetName, childDelta=True, config=None):
             numpy.put(self.FFD.coef[:, 1], self.ptAttachInd, temp[:, 1])
             numpy.put(self.FFD.coef[:, 2], self.ptAttachInd, temp[:, 2])
 
+        # Now add in the section local DVs
```

Comment:
done

No particular reason. I saw that Nick had put section local before local, so I though I'd follow suit and append it to the front. I not sure it matters since all the local variables use += when setting the coef of the FFD in their respective __call__ methods.